package.json

```diff
@@ -230,12 +230,12 @@
 		"postinstall": "node ./node_modules/vscode/bin/install"
 	},
 	"dependencies": {
-		"vscode-debugadapter": "^1.11.0",
-		"vscode-debugprotocol": "^1.11.0",
+		"vscode-debugadapter": "^1.12.0",
```

Comment:
Do you know whether this requires a certain version of VSCode? In my terminal branch I updated these along with the `engine/vscode` property but I couldn't find any info on what actually happened if people were using old versions of Code.